backend/db_access/urls.py

```diff
@@ -2,10 +2,12 @@
 from . import views
 
 urlpatterns = [
-    path("delivery/", views.getDeliveries),
-    path("brand/", views.getBrands),
-    path("transaction/", views.getTransactions),
```

Comment:
The /create urls and methods are now redundant because the new method that handle both POST and GET do the same. Therefore the /create urls and methods should be removed. Be aware: `delivery/create/` is used in `delivery-editor.tsx`.